internal/httputils/requests.go

```diff
@@ -0,0 +1,16 @@
+package httputils
+
+import "net/http"
+
+// RequestURI returns the path and query string (if present) from the request
+// For example: `/foo` or `/foo?hello=world`
+func RequestURI(r *http.Request) string {
```

Comment:
This method is maintain compatibility with the (undocumented...) behavior of fasthttp's `ctx.RequestURI`